index.js

```diff
@@ -1,110 +1,107 @@
+  const pullRequests = payload.issue.pull_request;
+  if (!pullRequests) {
+    console.log('It isn\'t a pull request.');
     return;
   }
-
-  const commentBody = payload['comment']['body'].toLowerCase();
-
-  if (commentBody.match('@seanbot') && commentBody.match('bump')) {
-
-    function getMasksFromCommentBody() {
-      let MAJOR = 0;
-      let MINOR = 0;
-      let PATCH = 0;
-      if (commentBody.match('major')) {
-        MAJOR = 1;
-      } else if (commentBody.match('minor')) {
-        MINOR = 1;
-      } else if (commentBody.match('patch')) {
-        PATCH = 1;
-      } else {
-        return;
-      }
-      const masks = [MAJOR, MINOR, PATCH];
-      return masks;
-    }
-
-    function getMsgFormPrUrl() {
-      const list = pullRequest['url'].split('/');
-      return {
-        user: list[4],
-        repo: list[5],
-        number: list[7]
-      };
-    }
-
-    const masks = getMasksFromCommentBody();
-    if (!masks) {
-      return;
-    }
-
-    github.pullRequests.get(getMsgFormPrUrl(), (err, res) => {
-      if (err) {
-        console.error(err && err.stack);
-        return;
-      }
-      const data = {
-        user: res['head']['repo']['owner']['login'],
-        repo: res['head']['repo']['name'],
-        path: 'package.json',
-        ref: res['head']['ref']
-      };
-
-      github.repos.getContent(data, (err, res) => {
-        if (err) {
-          console.error(err && err.stack);
-          return;
-        }
-
-        const content = res['content'];
-        const sha = res['sha'];
-        let _package = JSON.parse(new Buffer(content, 'base64').toString());
-        _package.version = bumpVersion(masks, _package.version);
-
-        github.repos.updateFile({
-          user: data['user'],
-          repo: data['repo'],
-          path: 'package.json',
-          branch: data['ref'],
-          message: 'bump version to ' + _package.version,
-          content: new Buffer(JSON.stringify(_package, null, '  ')).toString('base64'),
-          sha: res.sha
-        }, (err, res) => {
-          if (err) {
-            console.log(err && err.stack);
-            return;
-          }
-          console.log(res);
-        });
-      });
-    });
+  const commentBody = payload.comment.body.toLowerCase();
+  const masks = getMasksFromCommentBody(commentBody);
+  if (!commentBody.match('@seanbot') || !commentBody.match('bump' || !masks)) {
+    console.log(commentBody, 'It\'s not a bump action.');
+    return;
   }
+  getPullRequests(getMsgFormPrUrl(pullRequests.url))
+  .then((res) => {
+    const baseMsg = {
+      user: res.base.repo.owner.login,
+      repo: res.base.repo.name,
+      path: 'package.json',
+      ref: res.base.ref,
+    };
+    const headMsg = {
+      user: res.head.repo.owner.login,
+      repo: res.head.repo.name,
+      path: 'package.json',
+      ref: res.head.ref,
+    };
+    return Promise.all([getContent(baseMsg), getContent(headMsg)]);
+  })
+  .then((baseRes, headRes) => {
```

Comment:
`then` function should always return only 1 argument, so you should destructor them inside function like this:

```js
then((results) => {
  const {baseRes, headRes} = results;
  // ...
})
```